index-commercial.html

```diff
@@ -354,7 +354,7 @@
         </li>
         <li>
           <form action="https://help.openshift.com/hc/en-us/search" method="get" id="cse-search-form">
-            <input name="query" id="cse-search-input" class="navbar-search-query form-control" type="text" placeholder="Search" tabindex="1" autocomplete="off" autofocus>
+            <input name="query" id="cse-search-input" class="navbar-search-query form-control" type="text" placeholder="Search" tabindex="1" autocomplete="off" autofocus="autofocus">
```

Comment:
Remove aufocous from this form input, leave it on the other.

Autofocus automatically sets the focus on a specific form input when the page loads. If you're going to use this for whatever reason, you should only autofocus onto a single form field input (you can't have the cursor focus on two fields).